packages/cpp/data/c.tmLanguage.json

```diff
@@ -0,0 +1,1955 @@
+{
+	"information_for_contributors": [
+		"This file has been converted from https://github.com/atom/language-c/blob/master/grammars/c.cson",
+		"If you want to provide a fix or improvement, please create a pull request against the original repository.",
+		"Once accepted there, we are happy to receive an update request."
+	],
+	"version": "https://github.com/atom/language-c/commit/9c0c5f202741a5647025db8d5df5fefba47b036c",
```

Comment:
do we require to add copyright info for the grammars?

Reply:
They have pointers to their original location (incl license). We do the same as VSCode does.